app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,5 @@
+module CommentsHelper
+  def owner_cmt comment
```

Comment:
đối với các method return kiểu true/false nên đặt lại tên cho phù hợp nhé
```suggestion
  def is_owner_cmt? comment
```